module-2/backend/week_one.md

```diff
@@ -5,30 +5,89 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+ANSWER:
+GET, PUT, POST, PATCH, DELETE
+
 2. What is Sinatra?
+ANSWER:
+It's a framework that understands Ruby and makes easier to build web applications. It allows us to perform different actions and display a different view for each path that we define in the controller. It uses HTTP verbs and a very simplified syntax.
+
 4. What is MVC?
+ANSWER:
+Models/Viewer/Controller is a convention to structure your code when building a web application. The Models is reponsible for creating the Ruby objects (horse.rb), the Viewer takes care of the views that are rendered to the user for each path and the Controller handles the routes and the actions for each one.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+ANSWER:
+We follow conventions when writing code in general for readability, easy maintenance and keep improving it among all the members of the community. 
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+ANSWER:
+We can access instance variables that are created in the model (horse.rb) and are declared in the view(index.erb)
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
+ANSWER:
+Using the <% %> so we can write Ruby inside this tag in an erb file.
   ```ruby
   get '/horses' do
     erb :index
   end
   ```
-
+  
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+ANSWER:
+I would do:
+``` ruby
+  get '/horses' do
+  @horse = "Mr. Ed"
+    erb :index
+  end
+  ```
+And then in the index.erb, I'd put <%= @horse %>

```

Comment:
If we were going to use the local variable per the snippet above we could use `<%= name %>` without the `@` since it's a local instead of an instance variable.